source/preface.tex

```diff
@@ -0,0 +1,184 @@
+\begin{itemize}
+  \item explicit \grammarterm{template-head}s in generic lambdas
+  \item default construction and assignment of stateless closure types
+  \item lambda expressions in unevaluated operands
+  \item pack expansion of lambda \grammarterm{init-capture}s
+\end{itemize}
+\item generalized support for structured bindings
+\item support for inline namespaces in nested namespace definitions
+\item support for conditionally-explicit member functions
+\item extended support for class template argument deduction to cover
+aggregate initialization and alias templates
+\item reduced cases in which \keyword{typename} is required
+\item support for calling an undeclared \grammarterm{template-id}
+via argument-dependent name lookup
+\item relaxed access checking rules in template specialization declarations
+\item revised memory model
+\item expanded cases in which returned or thrown variables are implicitly moved
+\item extended support for variadic macros with \mname{VA_OPT}
+% library changes
+\item feature test macros and \libheaderref{version} header
+\item restricted valid uses of standard library functions and function templates
+\item addition of ranges and \libheaderref{ranges} header
+\item addition of calendar and time zone support
+\item addition of text formatting library and \libheaderref{format} header
+\item addition of synchronization facilities for
+waiting, notifications, semaphores, latches, and barriers, and
+\libheaderref{barrier}, \libheaderref{latch} and \libheaderref{semaphore} headers
+\item addition of mathematical constants library and \libheaderref{numbers} header
+\item support for representing source locations and \libheaderrefx{source_location}{source.location.syn} header
+\item addition of \tcode{span} view and \libheaderref{span} header
+\item addition of joining thread class and \libheaderrefx{stop_token}{thread.stoptoken.syn} header
+\item extensions to atomic types and operations, including
+\begin{itemize}
+  \item new class template \tcode{atomic_ref}
+  \item new atomic floating-point operations
+  \item new atomic smart pointer types
+  \item support for compare-and-exchange operations on types with padding
+  \item conversion of \tcode{memory_order} to a scoped enumeration
+\end{itemize}
+\item addition of \tcode{unsequenced} execution policy
+\item new utility functions, types, and templates in the standard library, including
+\begin{itemize}
+  \item \raggedright new type traits
+    \tcode{is_bounded_array},
+    \tcode{is_corresponding_member},
+    \tcode{is_layout_compatible},
+    \tcode{is_nothrow_convertible},
+    \tcode{is_pointer_interconvertible_base_of},
+    \tcode{is_pointer_interconvertible_with_class}, and
+    \tcode{is_unbounded_array}
+  \item new transformation traits
+    \tcode{remove_cvref},
+    \tcode{type_identity},
+    \tcode{unwrap_ref_decay}, and
+    \tcode{unwrap_reference}
+  \item new standard library primitive functions \tcode{assume_aligned} and
```

Comment:
(Is "primitive" a good word for this?)

Reply:
Ehh. I was thinking "things that can't be implemented in the language", but perhaps that's just not an interesting distinction to be drawing here. Simplified.